test/smoke/src/areas/positron/layouts/layouts.test.ts

```diff
@@ -0,0 +1,172 @@
+				await layouts.enterLayout('side_by_side');
+
+				// ------ Panel and Sidebar -------
+				// Both should be collapsed.
+				await expect(layouts.panelContent).not.toBeVisible();
+				await expect(layouts.sidebar).not.toBeVisible();
+
+				// ------ Auxiliary Bar -------
+				await expect(layouts.auxBar).toBeVisible();
+
+				// First view should be the session view, second should be terminal
+				await expect(layouts.auxBarViewsTab.nth(0)).toHaveText('Session');
+				await expect(layouts.auxBarViewsTab.nth(1)).toHaveText('Terminal');
+
+				const consoleSection = layouts.auxBar.getByLabel(/console section/i);
+				const variablesSection = layouts.auxBar.getByLabel(/variables section/i);
+				const plotsSection = layouts.auxBar.getByLabel(/plots section/i);
+				await expect(consoleSection).toBeVisible();
+				await expect(variablesSection).toBeVisible();
+				await expect(plotsSection).toBeVisible();
+
+				// Only the console section should be expanded
+				await expect(consoleSection).toHaveAttribute('aria-expanded', 'true');
+				await expect(variablesSection).toHaveAttribute('aria-expanded', 'false');
+				await expect(plotsSection).toHaveAttribute('aria-expanded', 'false');
+
+				// Should be in vertical order of console, variables, plots
+				const consoleY = await layouts.boundingBoxProperty(consoleSection, 'y');
+				const variablesY = await layouts.boundingBoxProperty(variablesSection, 'y');
+				const plotsY = await layouts.boundingBoxProperty(plotsSection, 'y');
+				expect(consoleY).toBeLessThan(variablesY);
+				expect(variablesY).toBeLessThan(plotsY);
+			});
+		});
+
+		describe('Notebook Layout', () => {
+
+			it('Verify notebook layout puts stuff in appropriate places', async function () {
+
+				const app = this.app as Application;
+				const layouts = app.workbench.positronLayouts;
+
+				// Enter layout with help pane docked in session panel
+				await layouts.enterLayout('notebook');
+
+				// ------ Panel -------
+				// Panel should be collapsed
+				await expect(layouts.panelContent).not.toBeVisible();
+
+				// In the collapsed layout, the first tab should be the console and the second
+				// should be the terminal
+				await expect(layouts.panelViewsTab.nth(0)).toHaveText('Console');
+				await expect(layouts.panelViewsTab.nth(1)).toHaveText('Terminal');
+
+				// When expanding the panel it should go to roughly 40% the height of the window
+				await layouts.panelExpandButton.click();
+				const fullAppHeight = await layouts.boundingBoxProperty(layouts.fullApp, 'height');
+				const panelHeight = await layouts.boundingBoxProperty(layouts.panel, 'height');
+				expect(panelHeight).toBeGreaterThan(fullAppHeight * 0.3);
+
+
+				// ------ Sidebar -------
+				// If the sidebar is collapsed is dependent upon the size of the window. If it
+				// is _not_ collapsed, then it needs to be wider than 180px.
+				// TODO: Test this after resizing the window to make sure behavior is correct.
```

Comment:
just wanted to double check if this TODO was supposed to have remained for future or if you had already done this check.

Reply:
Was a todo for future when we add functionality to change viewport size for test run.